migrations/2025-08-04-143048_add-VoidedPostCharge/up.sql

```diff
@@ -0,0 +1,6 @@
+-- Your SQL goes here
+ALTER TYPE "IntentStatus" ADD VALUE IF NOT EXISTS 'cancelled_post_capture';
+
+ALTER TYPE "AttemptStatus" ADD VALUE IF NOT EXISTS 'voided_post_charge';
+
+ALTER TYPE "EventType" ADD VALUE 'payment_cancelled_post_capture';
```

Comment:
Nit: Can you add the `IF NOT EXISTS` part as well for the `EventType` alter query?